images/stylelint/package.json

```
@@ -1,12 +1,12 @@
 {
   "dependencies": {
-    "stylelint": "13.13.1",
+    "stylelint": "14.6.1",
     "stylelint-config-prettier": "9.0.3",
-    "stylelint-config-recommended": "5.0.0",
-    "stylelint-config-recommended-scss": "4.3.0",
-    "stylelint-config-standard": "22.0.0",
-    "stylelint-order": "4.1.0",
+    "stylelint-config-recommended": "7.0.0",
+    "stylelint-config-recommended-scss": "6.0.0",
+    "stylelint-config-standard": "25.0.0",
```

[suggest] What if adding [stylelint-config-standard-scss](https://github.com/stylelint-scss/stylelint-config-standard-scss) also?

```suggestion
    "stylelint-config-standard": "25.0.0",
    "stylelint-config-standard-scss": "3.0.0",
```

---

I added it, but have to admit that I don't really know the role of stylelint configs and plugins. 

If we want our tool to continue checking LESS, SCSS, SSS and SASS formats, do we have to add the corresponding packages to package.json and plugins/configs to stylelint config file?

Currently we have only package for scss in the package.json file.

---

`images/stylelint/package.json` provides our default package set. It means users can have just their Stylelint configuration file (such as `.stylelintrc.yml`). For example:

```yaml
# This file is a `.stylelintrc.yml` file of a user.
extends: "stylelint-config-standard-scss"
```

In this case, Sider will analyze the user's code with our pre-installed `stylelint-config-standard-scss` and the user's `.stylelintrc.yml`. That user doesn't need to install the package. Having only the YAML file is enough.

I think our pre-installed package list should consist of packages widen-used or officially-maintained. The Stylelint configs or plugins for SCSS have been maintained by the Stylelint team, while other configs or plugins for LESS or SSS etc. are not.